services/checks/checks_test.go

```diff
@@ -38,12 +38,13 @@ import (
 
 	"github.com/percona/pmm-managed/models"
 	"github.com/percona/pmm-managed/services"
+	"github.com/percona/pmm-managed/utils/platform"
 	"github.com/percona/pmm-managed/utils/testdb"
 )
 
 const (
-	devChecksHost      = "check-dev.percona.com"
-	devChecksPublicKey = "RWTg+ZmCCjt7O8eWeAmTLAqW+1ozUbpRSKSwNTmO+exlS5KEIPYWuYdX"
+	devPortalAddress   = "https://check-dev.percona.com"
+	devPortalPublicKey = "RWTg+ZmCCjt7O8eWeAmTLAqW+1ozUbpRSKSwNTmO+exlS5KEIPYWuYdX"
```

Comment:
Keys should not be present in source, could you please lmk the reason this is present ?

Reply:
We have hardcoded public Percona Platform keys that we use for signatures verification of files that we receive from Portal. https://github.com/percona/pmm-managed/blob/05124a1d4ffae221ddd9d974e2c9f5d926f4afb2/utils/signatures/signatures.go#L27-L31

In tests we want to use dev environment instead of production, so we overriding prod keys with dev key. That let everyone to run this tests and check that everything is fine.

Reply:
This really needs to move to a mechanism for key management, hard-coding them in place, is fine for now, being as they are public keys based on your inclusion above, having multiple such keys is going to lead to manageability issues if nothing else.